### PR TITLE
add erlfmt, run on all files

### PR DESCRIPTION
**Pull request by evanmcc**
open for comments

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -1,39 +1,47 @@
...
+
+{project_plugins, [
+    erlfmt,
+    {rebar3_eqc, "1.3.0"}
+]}.
+
+{erlfmt, [
+    %% verbose,
+    write,
+    {print_width, 100}
```
Comment (JayKickliter) [inline rebar.config:39]: I'm not a fan of specifying this. If you want to format a single file with the `escript` version of `erlfmt`, e.g. via an editor command, this setting doesn't get picked up and the reformatted results may differ from what `rebar3 fmt` would do.

Comment (JayKickliter) [reply rebar.config:39]: (I admit this is a ridiculous state of affairs compared to say `clang-format` with its `some-repo/.clang-format` file)

Comment (evanmcc) [reply rebar.config:39]: I guess my hope here was to have this be in a pre-commit hook or something like that, so we wouldn't have to specify it.  `rebar3 fmt` takes less than a second on my machine though, so maybe a single-file version isn't needed?

Comment (Vagabond) [reply rebar.config:39]: I believe rebar3 fmt can also be told to crash if the file is not formatted, so we could make that part of CI

**Pull request closed without merging**